mmf/utils/distributed.py

```diff
@@ -154,12 +171,10 @@ def object_to_byte_tensor(obj, max_size=4094):
     return byte_tensor
 
 
-def byte_tensor_to_object(byte_tensor, max_size=4094):
+def byte_tensor_to_object(byte_tensor):
```

Comment:
Why remove the assert? Either keep it to max_size = MAX_SIZE_LIMIT. This is backwards incompatible change.

Reply:
Thanks! This is removed as byte-to-object does not need to be restricted by a max size. As long as it is a valid pickle encoded byte that can be hold by the `byte_tensor`, it is certainly not overflowing in size.

Reply:
I understand that, but we need to keep the variable for backwards compatibility.

Reply:
(changed to `max_size=MAX_SIZE_LIMIT`)